matching-core/pom.xml

```diff
@@ -40,9 +40,9 @@
             <version>${log4j.version}</version>
         </dependency>
         <dependency>
-            <groupId>com.graphhopper.external</groupId>
+            <groupId>com.bmw.hmm</groupId>
```

Comment:
I would like to provide the hmm-lib 1.0.0 as is. Changing the groupId afterwards (preferably after this pull request) is up to you.